content/en/about/feature-stages/index.md

```diff
@@ -69,18 +69,18 @@ Below is our list of existing features and their current phases. This informatio
 
 | Feature           | Phase
 |-------------------|-------------------
-| [Deny Checker](/docs/reference/config/policy-and-telemetry/adapters/denier/)         | Stable
-| [List Checker](/docs/reference/config/policy-and-telemetry/adapters/list/)        | Stable
-| [Pluggable Key/Cert Support for Istio CA](/docs/tasks/security/citadel-config/plugin-ca-cert/)        | Stable
 | [Service-to-service mutual TLS](/docs/concepts/security/#mutual-tls-authentication)         | Stable
 | [Kubernetes: Service Credential Distribution](/docs/concepts/security/#pki)   | Stable
+| [Pluggable Key/Cert Support for Istio CA](/docs/tasks/security/citadel-config/plugin-ca-cert/)        | Stable
+| [Authorization](/docs/concepts/security/#authorization)   | Beta
+| [End User (JWT) Authentication](/docs/concepts/security/#authentication)  | Beta
+| [Automatic mutual TLS](/docs/tasks/security/authentication/auto-mtls/) | Beta
 | [VM: Service Credential Distribution](/docs/concepts/security/#pki)         | Beta
-| [Mutual TLS Migration](/docs/tasks/security/authentication/mtls-migration)    | Beta
 | [Cert management on Ingress Gateway](/docs/tasks/traffic-management/ingress/secure-ingress-sds) | Beta
```

Comment:
```suggestion
| [Cert management on Ingress Gateway](/docs/tasks/traffic-management/ingress/secure-ingress-sds) | Stable
```

Reply:
And to keep the table ordered by phase, that would have to move up..

Reply:
Done.